app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,23 @@
+class PasswordResetsController < ApplicationController
+  def create
+    user = User.find_by_token(params[:token])
+    if user
+      user.password = params[:password]
+      user.generate_token
```

Comment:
So this would require creating a job to clear out tokens in the database if they were a certain age correct?